tools/src/main/java/com/nedap/archie/adl14/DefaultMultiplicityRemover.java

```diff
@@ -3,22 +3,37 @@
 import com.nedap.archie.adlparser.modelconstraints.BMMConstraintImposer;
 import com.nedap.archie.aom.Archetype;
 import com.nedap.archie.aom.CAttribute;
+import com.nedap.archie.aom.CAttributeTuple;
+import com.nedap.archie.aom.CComplexObject;
 import com.nedap.archie.aom.CObject;
 import com.nedap.archie.base.MultiplicityInterval;
 import com.nedap.archie.rminfo.MetaModels;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Removes any non-default cardinality or existence. Useful in ADL 1.4 to ADL 2 conversion, but could also
  * be used in archetype modeling tools.
  */
 public class DefaultMultiplicityRemover {
 
-
     private final MetaModels metaModels;
     private BMMConstraintImposer constraintImposer;
 
+    private boolean removeEmptyAttributes = false;
+
     public DefaultMultiplicityRemover(MetaModels metaModels) {
+        this(metaModels, false);
+    }
+
+    public DefaultMultiplicityRemover(MetaModels metaModels, boolean removeEmptyAttributes) {
```

Comment:
It is sort of separate, in the sense that there is an order dependency:
- first the default multiplicities must be removed
- then the attributes that now could have become empty must be removed

So it could be moved to a separate class, but then the API should still be a single call I think

Reply:
Instead I could rename the class to DefaultMultiplicitiesAndAttributesRemover, or perhaps 'DefaultRmStructureRemover'?

Reply:
DefaultRmStructureRemover sounds nice.